src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -66,7 +66,7 @@ def refresh_repo(self):
         self.invoke_package_manager(self.repo_refresh)
 
     # region Get Available Updates
-    def invoke_package_manager(self, command):
+    def invoke_package_manager(self, command, raise_on_exception=True):
```

Comment:
def invoke_package_manager_advanced(self, command, raise_on_exception=True):
     # full function body with additional behavior
.. return out, code

def invoke_package_manager(self, command):      # backwards compatible for 99% of cases
   out, code = self.invoke_package_manager_advanced(command, raise_on_exception=True)
   return out